Common/include/option_structure.hpp

```diff
@@ -952,19 +950,59 @@ inline TURB_FAMILY TurbModelFamily(TURB_MODEL model) {
     case TURB_MODEL::SA_E_COMP:
       return TURB_FAMILY::SA;
     case TURB_MODEL::SST:
-    case TURB_MODEL::SST_SUST:
       return TURB_FAMILY::KW;
   }
   return TURB_FAMILY::NONE;
 }
 
+/*!
+ * \brief SST Options
+ */
+enum class SST_OPTIONS {
+  NONE,        /*!< \brief No SST Turb model. */
+  V1994,       /*!< \brief 1994 Menter k-w SST model. */
+  V2003,       /*!< \brief 2003 Menter k-w SST model. */
+  MODIFIED,    /*!< \brief Modified k-w SST model. */
+  SUST,        /*!< \brief Menter k-w SST model with sustaining terms. */
+  VORTICITY,   /*!< \brief Menter k-w SST model with vorticity production terms. */
+  KL,          /*!< \brief Menter k-w SST model with Kato-Launder production terms. */
+  UNCERTAINTY, /*!< \brief Menter k-w SST model with uncertainty quantification modifications. */
+  RC,          /*!< \brief Menter k-w SST model with rotation/curvature corrections. */
+};
+static const MapType<std::string, SST_OPTIONS> SST_Options_Map = {
+  MakePair("NONE", SST_OPTIONS::NONE)
+  MakePair("V1994", SST_OPTIONS::V1994)
+  MakePair("V2003", SST_OPTIONS::V2003)
+  MakePair("MODIFIED", SST_OPTIONS::MODIFIED)
+  MakePair("SUSTAINED", SST_OPTIONS::SUST)
+  MakePair("VORTICITY", SST_OPTIONS::VORTICITY)
+  MakePair("KATO-LAUNDER", SST_OPTIONS::KL)
+  MakePair("UNCERTAINTY", SST_OPTIONS::UNCERTAINTY)
+  MakePair("CURVATURE", SST_OPTIONS::RC)
+};
+
+/*!
+ * \brief Structure containing parsed SST options.
+ */
+struct SST_ParsedOptions {
+  std::string sst_string = "Menter's k-w SST"; /*!< \brief Initial string for SST problems. */
```

Comment:
remove this string member please, the objective for the struct is to be light weight, you can still produce the string while parsing the options, it only needs to be printed by CConfig right?